api/v1/routes/blog.py

```diff
@@ -84,3 +86,19 @@ async def update_blog(id: str, blogPost: BlogRequest, db: Session = Depends(get_
         "message": "Blog post updated successfully",
         "data": {"post": jsonable_encoder(updated_blog_post)}
     }
+    
+@blog.delete("/{id}", status_code=status.HTTP_200_OK)
+def delete_blog_post(id: str, db: Session = Depends(get_db), current_user: User = Depends(get_super_admin)):
+    print("Deleting.........")
+    if not current_user:
+        return {"status_code":403, "message":"Unauthorized User"}
+    post = db.query(Blog).filter_by(Blog.id == id, Blog.is_deleted == False).first()
+    
+    if not  post:
+        return {"status_code":status.HTTP_404_NOT_FOUND, "message": "Blog with the given ID does not exist"}
+    
+    post.is_deleted = True
+    # db.delete(post)
+    db.commit()
+    
+    return {"message": "Blog post deleted successfully", "status_code": 200}
```

Comment:
remove print statements

don't make db query in routes folder, create a function for this query in services folder and make db request there

Reply:
Fixed